packages/igx-templates/igx-ts/dock-manager/default/files/src/app/__path__/__filePrefix__.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { <%=ClassName%>Component } from './<%=filePrefix%>.component';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { IgcDockManagerLayout, IgcDockManagerPaneType, IgcSplitPaneOrientation } from 'igniteui-dockmanager';
+
+describe('<%=ClassName%>Component', () => {
+  let component: <%=ClassName%>Component;
+  let fixture: ComponentFixture<<%=ClassName%>Component>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [<%=ClassName%>Component],
+      imports: [IgcDockManagerLayout, IgcDockManagerPaneType, IgcSplitPaneOrientation, NoopAnimationsModule]
```

Comment:
Umm not sure where this came from, I doubt the Dock manager being a Web Component exports any compatible `Ng-` decorated classes to import. Shouldn't this include the custom schema too?

PS: Indeed, this throws:
```
ERROR in src/app/dock-manager/dock-manager.component.spec.ts:14:17 - error TS2693: 'IgcDockManagerLayout' only refers to a type, but is being used as a value here.

14       imports: [IgcDockManagerLayout, IgcDockManagerPaneType, IgcSplitPaneOrientation, NoopAnimationsModule]
```